src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -46,19 +32,17 @@ namespace stan {
       }
 
       void set_max_delta(double d) {
-        max_delta_ = d;
+        max_deltaH_ = d;
       }
 
       int get_max_depth() { return this->max_depth_; }
-      double get_max_delta() { return this->max_delta_; }
+      double get_max_delta() { return this->max_deltaH_; }
 
       sample transition(sample& init_sample,
                         interface_callbacks::writer::base_writer& writer) {
         // Initialize the algorithm
         this->sample_stepsize();
 
-        nuts_util util;
-
         this->seed(init_sample.cont_params());
```

Comment:
I know this isn't new code, but why all the `this -> `?  Is there an issue with just calling `seed(...)` and other functions directly?  (I seem to recall something like that happening that we couldn't figure out).